### PR TITLE
Add gas refuel sum limit

### PR DESCRIPTION
**Pull request by ZhangTao1596**

### CODE REVIEW

File: src/neo/SmartContract/ApplicationEngine.cs [hunk 1]
```diff
@@ -85,6 +86,7 @@ public partial class ApplicationEngine : ExecutionEngine
         /// GAS spent to execute.
         /// </summary>
         public long GasConsumed { get; private set; } = 0;
+        public long GasRefueled { get; private set; } = 0;
```
Comment (shargon) [inline src/neo/SmartContract/ApplicationEngine.cs:89]: Add summary?

**Comment by erikzhang** [comment]
I've made the changes in #2561

File: src/neo/SmartContract/ApplicationEngine.cs [hunk 2]
```diff
@@ -86,7 +86,7 @@ public partial class ApplicationEngine : ExecutionEngine
         /// GAS spent to execute.
         /// </summary>
         public long GasConsumed { get; private set; } = 0;
-        public long GasRefueled { get; private set; } = 0;
+        public long GasRefueledSummary { get; private set; } = 0;
```
Comment (shargon) [inline src/neo/SmartContract/ApplicationEngine.cs:89]: I mean //</summary>

Comment (ZhangTao1596) [reply src/neo/SmartContract/ApplicationEngine.cs:89]: I see.  This is closed. Let's discuss in 2561.

**Pull request closed without merging**